src/vue/modules/movements-history/store/index.spec.js

```diff
@@ -0,0 +1,218 @@
+import { mutations, getters, actions } from './index'
+import { types } from './types'
+
+import { Wallet } from '@tokend/js-sdk'
+
+import { Movement } from '../wrappers/movement'
+import { Balance } from '../wrappers/balance'
+
+import * as Api from '../_api'
+import accountBalancesJSON from '@/test/mocks/account-balances'
+
+describe('movements-history.module', () => {
+  const accountId = 'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ'
+  const balances = [
+    {
+      id: 'BDPFDXJAL6UY53L52NNWPD7RTAO4EVZL55SWHNYVYJQ44BOEIQKL4FOJ',
+      asset: {
+        id: 'BTC',
+        type: 'assets',
+      },
+    },
+  ]
+  const movements = [
+    {
+      id: '176093659138',
```

Comment:
JSON API requires both `type` and `id` present in resource. It's better to follow the spec even in mocks

Reply:
Added type